ssl/statem/statem_lib.c

```diff
@@ -1068,12 +1068,21 @@ WORK_STATE tls_finish_handshake(SSL *s, WORK_STATE wst, int clearbufs, int stop)
                 dtls1_start_timer(s);
             }
         } else {
-            /*
-             * In TLSv1.3 we update the cache as part of processing the
-             * NewSessionTicket
-             */
-            if (!SSL_IS_TLS13(s))
+            if (SSL_IS_TLS13(s)) {
+                /*
+                 * We should only aim to use TLSv1.3 tickets once, so we remove
```

Comment:
"only aim" sounds like it's a goal, not something we forcibly do.  That confuses me.

Reply:
I'm with @richsalz 

_We only use..._

Reply:
Well, strictly speaking it *is* a goal and not something we forcibly do. This is about what goes in the client cache. To encourage people to only use tickets once we remove tickets from the cache once they are used. However, if an application attempts to reuse a ticket anyway then we don't attempt to detect that or prevent it. I've modified the comment a little to try and make that clearer.

Reply:
I think the new version is a lot more understandable.  Thanks.